### ISSUE TITLE
DirectX Decompilation Branch?

### ISSUE THREAD

**Issue by ZoriaRPG** [cuckydev/Cave-Story-Engine-2#86]
Is there a branch available with the pre-SDL2 conversion? There are a few things that I'd like to look at there, if possible.

I'm also a bit curious what the original-base 2D library was, if you happen to know.

**Comment by Clownacy** [comment]
Yeah, good timing actually - I just finished working on it a few days ago. [It's the 'accurate' branch on my repo.](https://github.com/Clownacy/Cave-Story-Engine-2/tree/accurate) The original game used DirectDraw.

**Comment by ZoriaRPG** [comment]
Beautiful. I'm the lead dev on an old Allegro 4 game engine (Zelda Classic) and one of our users pointed me at this project.

I was curious if the original was an Allegro game. We still use DirectDraw/DX, and are mostly stuck with it for the present. DXGL and Allegro Legacy are our two paths forward, as anything else is basically impossible. ;)
